Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -637,3 +561,17 @@ void EndJumpCheckDelay()
 	}
 	#endregion
 }
+
+public abstract class PhysicsState
+{
+	public abstract void Enter();
+	public abstract void Update();
+	public abstract void Exit();
+}
+
+public class DefaultState : PhysicsState
```

Comment:
`DefaultState` should go inside `ActorPhysics` the same way that the player's states go inside `ActorControls`.